tests/js/spec/components/dropdownReact.spec.jsx

```diff
@@ -153,4 +148,32 @@ describe('DropdownReact', function() {
       });
     });
   });
+
+  describe('Nested Dropdown', function() {
+    let wrapper;
+
+    beforeEach(function() {
+      if (wrapper) {
+        wrapper.unmount();
+      }
+
+      wrapper = mount(
+        <DropdownReact title="parent">
+          <DropdownReact title="nested" isNestedDropdown={true}>
+            Test
+          </DropdownReact>
+        </DropdownReact>
+      );
+    });
+
+    it('Opens / closes on mouse enter and leave', function() {
+      wrapper.find('a').simulate('click');
+      wrapper.find('.dropdown-menu a').simulate('mouseEnter');
+      expect(wrapper.find('.dropdown-menu').length).toBe(2);
+
+      wrapper.find('.dropdown-menu a').simulate('mouseLeave');
```

Comment:
why does this test work and the other one had to be deleted? they seems similar

Reply:
`mouseLeave` event is handled by the dropdown component and the `onClick` event isn't anymore